draft-ietf-privacypass-architecture.md

```diff
@@ -210,23 +210,28 @@ contexts:
 
 Redemption context:
 : The interactions and set of information shared
-between the Client and Origin. This context includes all information
-associated with redemption, such as the timestamp of the event, Client
-visible information (including the IP address), and the Origin name.
+between the Client and Origin, i.e., the information that is provided or
+otherwise available to the Origin during redemption that might be used
+to identify a Client. This context includes all information associated
+with redemption, such as the timestamp of the event, Client visible
```

Comment:
and the challenge too?

Reply:
The Origin constructs that, so I think it's sort of implicit, but we can add that too.